core-bundle/src/Resources/contao/forms/FormText.php

```diff
@@ -189,6 +191,9 @@ public function __get($strKey)
 					case 'url':
 					case HttpUrlListener::RGXP_NAME:
 						return 'url';
+
```

Comment:
The comment about https://github.com/contao/core/issues/5918 in L 171 should probably be adjusted/addressed as well.